### PR TITLE
Feature/mouse over

### PR DESCRIPTION
**Pull request by filipcynarski**

### CODE REVIEW

File: fluentlenium-core/src/main/java/org/fluentlenium/core/annotation/PageUrl.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package org.fluentlenium.core.annotation;
+
+import java.lang.annotation.Retention;
+
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+@Retention(RUNTIME)
+public @interface PageUrl {
```
Comment (Toilal) [inline fluentlenium-core/src/main/java/org/fluentlenium/core/annotation/PageUrl.java]: Missing javadoc

File: fluentlenium-core/src/main/java/org/fluentlenium/core/annotation/PageUrl.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package org.fluentlenium.core.annotation;
+
+import java.lang.annotation.Retention;
+
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
```
Comment (Toilal) [inline fluentlenium-core/src/main/java/org/fluentlenium/core/annotation/PageUrl.java]: Missing javadoc

File: fluentlenium-core/src/test/resources/index.html [hunk 3]
```diff
@@ -21,7 +21,7 @@
 <body style="" name="body">
 <h1>Heading</h1>
 
-<span id="oneline">A single line of text</span>
+<span id="oneline" onmouseover="javascript:document.getElementById('id').innerHTML='abc'">A single line of text</span>
```
Comment (Toilal) [inline fluentlenium-core/src/test/resources/index.html]: Maybe we coule keep this <span> unchanged, and add a new <span> for the purpose of this mouseover test.

**Comment by Toilal** [comment]
@filipcynarski it seems this branch also contains changes from PR #233. You should create a new branch from master, and cherry pick only the latest commit from this one to make the two pull requests independants.

File: fluentlenium-core/src/main/java/org/fluentlenium/core/action/MouseActions.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
+package org.fluentlenium.core.action;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Action;
+import org.openqa.selenium.interactions.Actions;
+
+public class MouseActions {
```
Comment (Toilal) [inline fluentlenium-core/src/main/java/org/fluentlenium/core/action/MouseActions.java]: I'm not sure this class really add value.

Comment (filipcynarski) [inline fluentlenium-core/src/main/java/org/fluentlenium/core/action/MouseActions.java]: I think it make sense to have separate class for Mouse and Keyboard Actions exactly like Selenium has. We are going to add D'n'D feature soon so this class will be extended shortly.

**Pull request closed without merging**